### PR TITLE
Merge Code From Forked Repo Of Team

### PR DESCRIPTION
**Pull request by lamnguyencse17**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  node: circleci/node@1.1.6
+jobs:
+  build-and-test:
+    executor:
+      name: node/default
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run: npm install
```
Comment (vcttai) [inline .circleci/config.yml:12]: @lamnguyencse17 
We should devide these code into other jobs, each job has one responsibility, e.g: unit-test-job, lint-job, ...

File: server/bin/www.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const bodyParser = require("body-parser");
+
+app.use(express.static("public"));
+app.use(bodyParser.json());
+app.use(express.urlencoded({ extended: true }));
+
+app.set("view engine", "ejs");
+app.set("views", "./views");
+
+app.get("/", function (request, response) {
```
Comment (vcttai) [inline server/bin/www.js:16]: @lamnguyencse17 
I think it's better to combine these routes into one: `/ctok`, `ftok`, `ftoc`
We can use only one route like this:
```
app.post("/convert", function (req, res) {
  const source = req.body.source;
  const target = req.body.target;

  return response.render("result", {
    source: source,
    target: target,
    value: converter.convert(parseFloat(request.body.temperature), source, target),
  });
});
```

Please consider, thanks!

File: server/utils/__tests__/converter.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const converter = require("../converter");
+
+test("convert F To C", function () {
```
Comment (vcttai) [inline server/utils/__tests__/converter.js:3]: @hieptien0512 

What if I had this case: `converter.convertFToC(-300)`?
When writing unit test, we should cover these cases:
  - Normal cases
  - Error cases
  - Corner cases

File: server/utils/converter.js [hunk 4]
```diff
@@ -1,12 +1,18 @@
-function convertToF(temperature) {
-  //TODO
+function convertFToK(temperature) {
+  return (temperature + 459.67) * (5 / 9);
 }
 
-function convertToC(temperature) {
-  //TODO
+function convertCToK(temperature) {
```
Comment (vcttai) [inline server/utils/converter.js:5]: @dcalau29 
What if I had this case: converter.convertCToK(-300)?

**Comment by lamnguyencse17** [comment]
Ok anh trong hôm nay tụi em sẽ giải quyết những vấn đề này
